### ISSUE TITLE
Many duplicate selectors

### ISSUE THREAD

**Issue by Faisal-Manzer** [postcss/postcss-dark-theme-class#5]
I don't know what's happening but it seems like there is duplicate selector being made. I guess to check for the pre-processed style is not there.

![Screenshot 2020-02-01 at 1 43 08 AM](https://user-images.githubusercontent.com/18066439/73571429-7eb71e00-4494-11ea-9e3e-4ff934400638.png)

**Comment by ai** [comment]
How do you use this plugin and what tool you use to compile CSS?

**Comment by Faisal-Manzer** [comment]
I am using it with `preact-cli`. Also, I am using `node-sass`.

```
# postcss.config.js

module.exports = {
    plugins: [
        require('postcss-viewport-height-correction'),
        require('postcss-normalize')({ forceImport: 'normalize.css' }),
        require('autoprefixer'),
        require('postcss-combine-media-query'),
        require('postcss-dark-theme-class')({ darkClass: 'dt', lightClass: 'lt' }),
        require('postcss-combine-duplicated-selectors')
    ]
};
```

**Comment by ai** [comment]
Do you call `postcss-cli` on already compiled CSS from previous step or you have separated `src/` and `dist/` folders?

**Comment by Faisal-Manzer** [comment]
Yeah, the Webpack config first converts `sass` to `CSS` then it processed by PostCSS. `preact-cli` is just like `create-react-app` so I don't know all those internal workings.

**Comment by ai** [comment]
1. What is the input CSS?
2. Can you disable postcss-combine-duplicated-selectors?
